util/certs.go

```diff
@@ -0,0 +1,57 @@
+package util
```

Comment:
I am not a fan of this `util` package. There should be separated packages doing one thing well.